README.md

```diff
@@ -189,6 +189,13 @@ $ pip install invoke==0.13.0
 $ invoke setup
 ```
 
+- Temporarily, tokumx needs to be installed separately from the invoke setup script:
+
+```bash
+$ tap chrisseto/homebrew-tokumx
```

Comment:
Er, yeah. Bad copy paste from tasks/__init__ where it does the brew part. Will fix that quick.